tests/common/ExecutionHelper.cs

```diff
@@ -134,6 +134,12 @@ public void ParseMessages ()
 				var line = l;
 				var msg = new ToolMessage ();
 				var origin = string.Empty;
+
+				if (l.Contains ("warning MT0099: Not linking with WatchKit because Xcode 11 beta 1 does not ship with the WatchKit framework")) {
```

Comment:
comment / FIXME to make reverting this easier

Reply:
It's already listed in #6212, but I'll add some text here too.